x-pack/plugins/security_solution/server/features.ts

```diff
@@ -341,20 +348,23 @@ const subFeatures: SubFeatureConfig[] = [
         privileges: [
           {
             api: [
+              'lists-all',
+              'lists-read',
+              'lists-summary',
               `${APP_ID}-writeHostIsolationExceptions`,
               `${APP_ID}-readHostIsolationExceptions`,
             ],
             id: 'host_isolation_exceptions_all',
             includeIn: 'none',
             name: 'All',
             savedObject: {
-              all: [],
+              all: [EXCEPTION_LIST_NAMESPACE_AGNOSTIC],
               read: [],
             },
             ui: ['writeHostIsolationExceptions', 'readHostIsolationExceptions'],
           },
           {
-            api: [`${APP_ID}-readHostIsolationExceptions`],
+            api: ['lists-read', 'lists-summary', `${APP_ID}-readHostIsolationExceptions`],
```

Comment:
Don't we also need to add `EXCEPTION_LIST_NAMESPACE_AGNOSTIC` to the `SavedObject` read list?  Same for the other artifact read privileges below where `'lists-read', 'lists-summary'` is needed for `api`?

Reply:
so I also thought that, but it turns out that it seems to work just fine without it. I'm not sure why thought 🤷‍♂️ 

For the mutating actions (`CUD`) it definitely need it.